===================================================================
--- astro
+++ vue
@@ -1,6 +1,6 @@
-<label class="ui-text-field" data-testid="email">
+<label class="ui-text-field">
   <span class="ui-label">Email</span>
   <span class="ui-field">
-    <input placeholder="name@example.com" required type="email">
+    <input data-testid="email" placeholder="name@example.com" required type="email">
   </span>
 </label>
